microprofile/tests/junit5/src/main/java/io/helidon/microprofile/tests/junit5/HelidonJunitExtension.java

```diff
@@ -458,6 +460,43 @@ private AddBeansExtension(Class<?> testClass, List<AddBean> addBeans) {
 
         @SuppressWarnings("unchecked")
         void registerOtherBeans(@Observes AfterBeanDiscovery event) {
+
```

Comment:
This is not the way to do what you want to do.
The correct event to handle is `ProcessInjectionPoint`, which will give you each injection point, where you can access the annotations of that injection point - so you do not need to use reflection at all, and you also do not read `Inject` annotation.

Reply:
Fixed